Jenkinsfile

```diff
@@ -18,7 +18,7 @@ pipeline {
 				wrap([$class: 'Xvnc', useXauthority: true]) {
 					sh """
 					mvn clean verify --batch-mode --fail-at-end -Dmaven.repo.local=$WORKSPACE/.m2/repository \
-						-Pbuild-individual-bundles -Ptest-on-javase-21 -Pbree-libs -Papi-check -Pjavadoc\
+						-Pbuild -Ptest-on-javase-21 -Pbree-libs -Papi-check -Pjavadoc\
```

Comment:
This change is wrong. Whatever is passed via maven.config would better be removed entirely here , aka this -Pbuild should be deleted as there is no such profile at all AFAICT.

Reply:
damn typo